sql-statements/sql-statement-select.md

```diff
@@ -10,82 +10,51 @@ The `SELECT` statement is used to read data from TiDB.
 
 ## Synopsis
 
-**SelectStmt:**
-
-![SelectStmt](/media/sqlgram/SelectStmt.png)
-
-**FromDual:**
-
-![FromDual](/media/sqlgram/FromDual.png)
-
-**SelectStmtOpts:**
+```ebnf+diagram
+SelectStmt ::=
+    ( SelectStmtBasic | SelectStmtFromDualTable | SelectStmtFromTable )
+    OrderBy? SelectStmtLimit? SelectLockOpt? SelectStmtIntoOption
 
-![SelectStmtOpts](/media/sqlgram/SelectStmtOpts.png)
+SelectStmtBasic ::=
+    "SELECT" SelectStmtOpts Field ("," Field)* ( "HAVING" Expression)?
 
-**SelectStmtFieldList:**
+SelectStmtFromDualTable ::=
+    "SELECT" SelectStmtOpts Field ("," Field)* "FROM" "DUAL" WhereClauseOptional
 
-![SelectStmtFieldList](/media/sqlgram/SelectStmtFieldList.png)
+SelectStmtFromTable ::=
+    "SELECT" SelectStmtOpts Field ("," Field)* "FROM" TableRefsClause WhereClauseOptional SelectStmtGroup HavingClause WindowClauseOptional
```

Comment:
AFAIK `("HAVING" Expression)?` and `HavingClause` are equivalent. The two references from `SelectStmtBasic` and `SelectStmtFromTable` should be using the same pattern.

Also I suggest *not* hiding the lone `?` behind `HavingClause` and `WhereClauseOptional` and `SelectStmtGroup` and `WindowClauseOptional` etc.